docs/content.zh/docs/deployment/ha/kubernetes_ha.md

```diff
@@ -31,6 +31,14 @@ Flink's Kubernetes HA services use [Kubernetes](https://kubernetes.io/) for high
 Kubernetes high availability services can only be used when deploying to Kubernetes.
 Consequently, they can be configured when using [standalone Flink on Kubernetes]({{< ref "docs/deployment/resource-providers/standalone/kubernetes" >}}) or the [native Kubernetes integration]({{< ref "docs/deployment/resource-providers/native_kubernetes" >}})
 
+## Prerequisites
+
+In order to use Flink's Kubernetes HA services you must fulfill the following prerequisites:
+
+- Kubernetes >= 1.9.
+- `default` service account with [RBAC]({{< ref "docs/deployment/resource-providers/native_kubernetes" >}}#rbac) permissions to create, edit, delete ConfigMaps.
```

Comment:
Maybe we should not guide the users to set the `default` service account here since any service account with enough permission will work. Also in the production environment, I think no one will use the `default` service account.

Another thing is that we need to note here that users should set the [ServiceAccount](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for JobManager deployment/job and TaskManager deployment manually in the yaml files.

Reply:
Alright, then let me update this PR a bit.